README.md

```diff
@@ -1,2 +1,31 @@
-# bootstrap-installer
-One-line installation of Bootstrap
+# bootstrap-installer
+
+![maintained](https://img.shields.io/badge/maintained-yes-success?style=flat-square)
+
+## Description
+
+This project contains a "one-line" script for enabling the installation of dependencies with the dedicated avengineers repository [bootstrap](https://github.com/avengineers/bootstrap).
+It is designed to simplify the process by using a Powershell script to clone the repository.
+This will then be used for setting up your environment by installing different dependencies.
+
+## How to use it
+
+Running `install.ps1` will clone the above mentioned repository into your current working directory.

```

Comment:
```suggestion
Running `install.ps1` will clone the above mentioned `bootstrap` repository to a directory `.bootstrap` in your current working directory.

```

Reply:
I incorporated this suggestion into the latest push